test/functional/collection_tests.js

```diff
@@ -1701,7 +1092,7 @@ describe('Collection', function() {
     });
   });
 
-  describe('countDocuments with mock server', function() {
+  describe('countDocuments with mock server', () => {
```

Comment:
I think this shouldn't be an arrow function.

Reply:
Fixed